static/js/subs.js

```diff
@@ -256,6 +256,11 @@ function settings_for_sub(sub) {
     return $("#subscription_settings_" + id);
 }
 
+function subscriber_count_for_sub(sub) {
```

Comment:
This function is I think named confusingly, since it sounds like it returns a number whereas it actually returns a jquery object where the count is displayed.

Further, since the function is only called to later set its value using subscribers_count.text(), I think the API would be a lot clearer if instead the function were e.g. `rerender_subscribers_count(sub, count)`, that actually updates the value.